### PR TITLE
(DOCS): Edit Create the first admin

### PR DESCRIPTION
**Pull request by paulofariadias**

### CODE REVIEW

File: administrator-guides/custom-fields/README.md [hunk 1]
```diff
@@ -1,60 +0,0 @@
-# Custom Fields
```
Comment (MartinSchoeler) [inline administrator-guides/custom-fields/README.md:1]: Are these changes related to this PR? If not, please remove these changes. Thanks!

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

**Comment by grahamsmith** [comment]
Closing this due to inactivity.
